tools/sigma/backends/lacework.py

```diff
@@ -62,6 +61,73 @@
           - EVENT_TIME
           - EVENT
         alertProfile: LW_CloudTrail_Alerts
+        evalFrequency: Hourly
+    product.categories:
+      linux.file_create:
+        evaluatorId:
+        source: LW_HE_FILES
+        conditions:
+          # evaluated hourly and file create time within the last hour
+          - and diff_minutes(FILE_CREATED_TIME, current_timestamp_sec()::timestamp) <= 60
+        fieldMap:
+          - sigmaField: TargetFilename
+            laceworkField: PATH
+            matchType: exact
+        returns:
+          - RECORD_CREATED_TIME
+          - MID
+          - PATH
+          - FILE_TYPE
+          - SIZE
+          - FILEDATA_HASH
+          - OWNER_UID
+          - OWNER_USERNAME
+          - FILE_CREATED_TIME
+        alertProfile: LW_HE_Files.HE_File_NewViolation
```

Comment:
i saw in the laceup-lpp-for-red-team channel Djay shared an updated list of alert profile names, i think this should now be `LW_HE_FILES_DEFAULT_PROFILE.HE_File_NewViolation`

Reply:
Fixed.